packages/ember-auto-import/ts/splitter.ts

```diff
@@ -17,7 +17,7 @@ const debug = makeDebug('ember-auto-import:splitter');
 const resolver = ResolverFactory.createResolver({
   // upstream types seem to be broken here
   fileSystem: new CachedInputFileSystem(new NodeJsInputFileSystem(), 4000) as unknown as AbstractInputFileSystem,
-  extensions: ['.js', '.json'],
+  extensions: ['.ts', '.js', '.json'],
```

Comment:
It's worth mentioning that the order is relevant here. The first match will be imported.

This means that _now_ , `.ts` files will take precedence. I think, we may want to put `.ts` last instead.